ts/Stock/Indicators/Chaikin/ChaikinIndicator.ts

```diff
@@ -73,9 +73,10 @@ class ChaikinIndicator extends EMAIndicator {
          * Paramters used in calculation of Chaikin Oscillator
          * series points.
          *
-         * @excluding index, period
+         * @excluding index
```

Comment:
If we're giving users the ability to change the `period`, we should inform them where this parameter is used.
Based on the `ChainkinIndicator.ts` this value is only used for `AD` calculation.
``` TS
        ADL = AD.prototype.getValues.call(this, series, {
            volumeSeriesID: params.volumeSeriesID,
            period: period
        }) as IndicatorValuesObject<TLinkedSeries>;

```
This information should be included in the API. My suggestion: 
```
Parameter used indirectly for calculating the accumulation/distribution indicator.  
Decides about the number of data points that are taken into account for the indicator calculations.
```